source/fundamentals/data-formats/codecs.txt

```diff
@@ -246,71 +222,55 @@ suppose you wanted to override the default provider behavior of a ``Codec`` for
 enum types with your custom ``MyEnumCodec``, you must add it to the registry
 list prior to the default codec registry as shown in the example below:
 
-.. code-block:: java
-   :emphasize-lines: 2
-
-   CodecRegistry newRegistry = CodecRegistries.fromRegistries(
-       CodecRegistries.fromCodecs(new MyEnumCodec()),
-       MongoClientSettings.getDefaultCodecRegistry());
+.. literalinclude:: /examples/generated/CodecTest.snippet.override-default-codec.kt
+   :language: kotlin
 
 For more information about the classes and interfaces in this section, see
 the following API documentation sections:
 
 - `CodecProvider <{+api+}/apidocs/bson/org/bson/codecs/configuration/CodecProvider.html>`__
 - `Default codec registry <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.html#getDefaultCodecRegistry()>`__
 
-
 BsonTypeClassMap
```

Comment:
Not sure I'd include this section at all - thoughts @jyemin ?

Reply:
Waiting on reply to make changes!